src/webui/webapplication.h

```diff
@@ -43,12 +43,10 @@
 #include "base/utils/net.h"
 #include "base/utils/version.h"
 
-constexpr Utils::Version<int, 3, 2> API_VERSION {2, 8, 2};
-
 class APIController;
 class WebApplication;
 
-constexpr char C_SID[] = "SID"; // name of session id cookie
+const Utils::Version<int, 3, 2> API_VERSION {2, 8, 3};
```

Comment:
Why did you move it down? Isn't it more important than forward declarations?

Reply:
In addition, by changing it from `constexpr` to `const`, you implicitly changed its linkage to static, so now this constant will be created in every compilation unit that includes this header. If you don't like `constexpr` here, you should use `inline` instead.

Reply:
>Why did you move it down? Isn't it more important than forward declarations?

The way I see is, header includes and forward declarations are more tightly related and although `API_VERSION` is important it is no more than just a variable. Let me know if you want it to stay at the old place.

Reply:
>Let me know if you want it to stay at the old place.

I'd like to see it as close to the top of header as possible, since it has a more global meaning than the other declarations here. Or even we should put it in a separate header.

Reply:
Reverted and addressed all other comments.